bentoml/cli/serverless.py

```diff
@@ -142,39 +80,38 @@ def check_serverless_compatiable_version():
             'Incompatiable serverless version, please install version 1.40.0 or greater')
 
 
-def generate_serverless_bundle(bento_service, platform, archive_path, output_path, extra_args):
-    TEMP_FOLDER_PATH = './temp_bento_serverless'
+def generate_serverless_bundle(bento_service, platform, archive_path, extra_args):
     check_serverless_compatiable_version()
+
+    provider = SERVERLESS_PROVIDER[platform]
+    TEMP_FOLDER_PATH = './temp_bento_serverless_' + datetime.now().isoformat()
```

Comment:
why not just use the ".bento-deployment" path here? that way we can avoid the 'save_deployment_archive' in the end, which is just copying the entire directory